sample-app/src/main/java/com/robustwealth/microservices/sample/repository/common/SqlReader.java

```diff
@@ -19,8 +19,8 @@ public SqlReader(Class<?> repositoryClass) {
 
     @Nonnull
     public String getSql(String queryName) {
-        final InputStream stream = getClass().getResourceAsStream(
-                File.separator + getBaseResourceFromRepositoryClassName() + "_" + queryName + ".sql");
+        final String resourceName = getBaseResourceFromRepositoryClassName() + "_" + queryName + ".sql";
```

Comment:
I think you are missing the initial file separator here.

Reply:
Initial separator did not affect the function because the method can take relative paths if the resource comes from the same class (which it does)

Reply:
I could see this since you are using the getClassLoader()'s version of the method.